python/oneflow/nn/modules/global_cast.py

```diff
@@ -75,16 +77,56 @@ def global_to_global_op(
 
 
 def to_global_op(input, placement=None, sbp=None, **kwargs):
-    assert isinstance(input, Tensor)
-
-    if input.is_global:
-        return global_to_global_op(input=input, placement=placement, sbp=sbp, **kwargs)
-    else:
-        if "grad_sbp" in kwargs:
-            del kwargs["grad_sbp"]
-        return local_to_global_op(input=input, placement=placement, sbp=sbp, **kwargs)
+    r"""Converts the input to global if it is local, otherwise performs designated placement and/or sbp conversion.
```

Comment:
Converts the input tensor or input tensor in List/Tuple/Dict to a global tensor.

Reply:
Refer to flollowing apis to refine the API Doc:
- https://oneflow.readthedocs.io/en/master/tensor.html?highlight=to_global#oneflow.Tensor.to_global
- https://oneflow.readthedocs.io/en/master/tensor.html?highlight=to_global#oneflow.Tensor.to_local

API doc should has an example.

Reply:
Corrected, example added